src/specific/s_input.c

```diff
@@ -666,7 +666,7 @@ static const char *S_Input_GetButtonName(SDL_GameControllerButton button)
                 case SDL_CONTROLLER_BUTTON_DPAD_DOWN:     return "\202 ";
                 case SDL_CONTROLLER_BUTTON_DPAD_LEFT:     return "\200 ";
                 case SDL_CONTROLLER_BUTTON_DPAD_RIGHT:    return "\201 ";
-                case SDL_CONTROLLER_BUTTON_MISC1:         return "SHARE"; /* Xbox Series X share button, PS5 microphone button, Nintendo Switch Pro capture button, Amazon Luna microphone button */
+                case SDL_CONTROLLER_BUTTON_MISC1:         return "SHARE";    /* Xbox Series X share button */
```

Comment:
Why the reduced comments?

Reply:
It's in a `// clang-format off` block so no. But, the line was really long, so I trimmed each console platform's comment to be relevant to that block. I left the full comment in the `default:` case. The comment itself is taken directly from SDL's codebase.

Reply:
OK I see, makes sense.